style.css

```diff
@@ -494,3 +501,172 @@ footer hr {
   border: 3px solid green;
   border-radius: 3px;
 }
+
+/* add style for desktop version */
+@media only screen and (min-width: 768px) {
+  /* add style for header of the page */
+  .nav-logo,
+  .humburger {
+    display: none;
+  }
+
+  .wrapper {
+    width: 1440px;
+    margin: 0 auto;
+    background: url("images/Header-llustration-desktop.svg");
+    background-position: top;
+    background-repeat: no-repeat;
+    gap: 60px;
+    display: flex;
+  }
+
+  header {
+    width: 100%;
+    align-self: center;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: center;
+  }
+
+  .nav-menu {
+    width: 18%;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .nav-menu li {
+    list-style: none;
+  }
+
+  .nav-menu li a {
+    text-decoration: none;
+    color: #344563;
+    font-size: 15px;
+    font-weight: 600;
+  }
+
+  /* add stye for section 1 on main part of page */
+  .section1 {
+    width: 60%;
+    align-self: center;
+    padding: 6% 10% 0;
+  }
```

Comment:
- [ ] Your project styling is fantastic, but I would like to make a suggestion to further improve it. Specifically, I recommend adjusting the positioning of the button with the arrow, moving it up to match the design in the Figma template more closely. Making this change will help ensure consistency in the design and improve the overall appearance of the project.

**Your design**

![Screenshot from 2023-04-05 08-44-59](https://user-images.githubusercontent.com/105117832/230100265-58ff0232-3b15-451d-acd5-ba3f39286e40.png)

**Figma**

![Screenshot from 2023-04-05 08-44-46](https://user-images.githubusercontent.com/105117832/230100344-f1794ee9-1a4f-4651-93d4-a4b1f67b8fff.png)